docs/organizations/roles-and-permissions-for-synced-organizations.md

```diff
@@ -208,7 +208,7 @@ See [managing people](managing-people.md) to list and manage the members of your
 
 ## Configuring who can change analysis configurations {: id="change-analysis-configuration"}
 
-By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues for your organization](../repositories/issues.md#ignoring-and-managing-issues) and [configure your code patterns](../repositories-configure/code-patterns.md) on your organization.
+By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues for your organization](../repositories/issues.md#ignoring-and-managing-issues), [ignore files](../repositories-configure/ignoring-files.md) and [configure your code patterns](../repositories-configure/code-patterns.md) on your organization.
 
 ![Configuring who can change analysis configurations](images/analysis-configuration.png)
```

Comment:
Can we have a narrower screenshot? :pray: Normally, I resize the window to make it as small as possible but without affecting the rendering of the several UI elements. It helps improve the readability of the screenshots, especially when browsing the documentation on a smaller screen.